### PR TITLE
samples: console: add print statements for user

### PR DESCRIPTION
**Pull request by nniranjhana**
Add print statements to let user know what kind
of input the test is waiting for.

Signed-off-by: Niranjhana N <niranjhana.n@intel.com>

### CODE REVIEW

File: samples/subsys/console/echo/src/main.c [hunk 1]
```diff
@@ -10,6 +10,8 @@ void main(void)
 {
 	console_init();
 
+	printk("Enter a character\n");
```
Comment (pfalcon) [inline samples/subsys/console/echo/src/main.c]: Please replace with: "Start typing characters to see them echoed back."

File: samples/subsys/console/getchar/src/main.c [hunk 2]
```diff
@@ -12,6 +12,8 @@ void main(void)
 {
 	console_init();
 
+	printk("Enter a character\n");
```
Comment (pfalcon) [inline samples/subsys/console/getchar/src/main.c]: Please replace with: "Start typing characters to see their codes printed."

File: samples/subsys/console/getline/src/main.c [hunk 3]
```diff
@@ -13,6 +13,8 @@ void main(void)
 {
 	console_getline_init();
 
+	printk("Enter a string\n");
```
Comment (pfalcon) [inline samples/subsys/console/getline/src/main.c]: Please replace with: "Enter a line".

**Review by pfalcon — CHANGES_REQUESTED** [review]
If we want to clarify it for user, let's have more detailed and unambiguous messages.

**Comment by nniranjhana** [comment]
Sure @pfalcon I have changed them to be more explicit, thanks

**Review by pfalcon — APPROVED** [review]
Thanks!

**Review by nashif — APPROVED** [review]

**Pull request merged**